lib/hon_graffiti_phoenix/parsers/markup_parser.ex

```diff
@@ -0,0 +1,64 @@
+defmodule HonGraffitiPhoenix.Parsers.MarkupParser do
+  alias HonGraffitiPhoenix.Parsers.DecoratedString
+
+  @color_map %{
+    "w" => "white", "*" => "white", "r" => "red",
+    "b" => "blue", "y" => "yellow", "m" => "magenta",
+    "n" => "brown", "p" => "purple", "o" => "orange",
+    "t" => "teal", "v" => "grey", "g" => "green",
+    "k" => "black", "" => "black"}
```

Comment:
Actually, I take that back, we need to test this in HoN, I'm not sure that that's the behavior that occurs... Either way, though, per your question about when to raise, I don't think we should as I don't think HoN would.